dipy/data/fetcher.py

```diff
@@ -592,16 +595,19 @@ def read_syn_data():
 """
 
 
-def read_mni_template(contrast="T2"):
+def read_mni_template(version="a", contrast="T1"):
```

Comment:
I know this PR is already merged, but isn't it a bad idea to silently change the default value of some parameter like this? Someone calling this from a script without any parameter might suddently encounter bizarre results...